shell/platform/linux/fl_view_private.h

```diff
@@ -37,12 +48,14 @@ void fl_view_add_gl_area(FlView* view,
  * @view: an #FlView.
  * @widget: a #GtkWidget.
  * @geometry: geometry of the widget.
+ * @mutations: (transfer-full): the mutations applied to the widget.
```

Comment:
This should be 'transfer full' (i.e. no hyphen). There seems to be a couple of existing cases in the code which are wrong.

Reply:
Fix for existing case: https://github.com/flutter/engine/pull/25964

Reply:
Actually, it would be better not to transfer ownership, but use a g_ptr_array_ref() inside the function(s).

Reply:
Done.